### PR TITLE
Changing the namspace.

### PR DESCRIPTION
**Pull request by Araks8**

### CODE REVIEW

File: magneto_lib/magneto.h [hunk 1]
```diff
@@ -3,10 +3,12 @@
 
 namespace cr {
 namespace magneto {
+namespace chessary {
 
 // Defines library API
 
 }  // namespace magneto
```
Comment (vgasparyan1995) [inline magneto_lib/magneto.h:10]: The order of the comments should be the backwards of the declarations, first the `chessary` namespace closes and then the `magneto`.

**Review by vgaspary — COMMENTED** [review]
Please correct the order of the namespace comments.

Do you think 3 levels make the code too long? Consider:

```
auto state = cr::magneto::chessary::CreateInitialState();
```

Of course one can always do `using namespace cr::magneto`, so that might not be a problem. Please share your opinion.

File: magneto_lib/magneto.h (hunk 1 above)
Comment (Araks8) [reply magneto_lib/magneto.h]: Done.

**Review by vgaspary — APPROVED** [review]

**Comment by Araks8** [comment]
I do think that 3 levels of code make the code too long, but I also think that it's better be long with clear meaning, than short and hard to understand for what those names stand for. Besides as you've mentioned one can always use `using namespace cr::magneto` or some other aliases.

**Comment by vgasparyan1995** [comment]
Won't merge.

**Pull request closed without merging**